websocketOutportDriver/payloadParser.go

```diff
@@ -105,50 +102,3 @@ func padUint32ByteSlice(initial []byte) []byte {
 	padding := bytes.Repeat([]byte{0}, 4)
 	return append(padding, initial...)
 }
-
-// PrepareArgsSaveBlock will prepare save block data
-func PrepareArgsSaveBlock(args outport.ArgsSaveBlockData) outport.ArgsSaveBlockData {
```

Comment:
do not forget to move this part that hex encodes the keys from the maps at the node level

Reply:
Yes. Good point